.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -46,4 +43,11 @@ If this PR follows standards of revertability, check the box below.
 Otherwise replace it with detailed instructions or reasons a rollback is impossible.
 -->
 
-- [ ] This PR can be reverted after deploy with no further considerations 
+- [ ] This PR can be reverted after deploy with no further considerations
+
+### Labels & Review
+- [ ] Risk label is set correctly
+- [ ] The set of people pinged as reviewers is appropriate for the level of risk of the change
+
+## Final self-reflection
+- [ ] My safety story above is convincing and gives me confidence that this PR will not introduce a regression
```

Comment:
Yeah, I think we have a disagreement/are at an impasse. For me an integral part of submitting a PR is thinking critically about what I'm doing and making sure it is safe and free of regressions before clicking submit. The extra comment you added for clarification actually makes it worse for me because it's even text to scan over and then realize it's redundant with my process and can be ignored. I'm happy to poll the team to find out if others find it useful. I will accept it if a majority of others find it useful.

Reply:
Sounds good, I won't add the comment. Based on the poll result I'm happy to accept a decisive team opinion over mine here as well.